Phase2.1DataExtractionOracle.sql

```diff
@@ -115,10 +115,10 @@ insert INTO patientsummary (
         race_collected
     )
 	select 'KUMC', c.patient_num, c.admission_date, 
-		round(sysdate - c.admission_date) ,
-		(case when trunc(a.last_discharge_date) = trunc(sysdate)  then TO_DATE('01/01/1900','mm/dd/rrrr') 
+		round(TO_DATE('09/10/2021','mm/dd/rrrr') - c.admission_date) ,
```

Comment:
Can you make it variable ? see https://github.com/kumc-bmi/PhaseX.2SqlDataExtraction/pull/1/files#diff-386484598bfdcab1ffb4a188426338d10ee96005abb14804138dd6275523c789R7 CE_PhaseX2_Files_oracle.sql:line7

Reply:
https://github.com/kumc-bmi/Phase2.1SqlDataExtraction/pull/3/commits/ad2af0841e034a469ab645ad6f5ff44fefef8f07